infra/gcp/terraform/kubernetes-public/k8s-infra-prow.tf

```diff
@@ -38,12 +38,6 @@ resource "google_service_account" "k8s_infra_prow" {
   display_name = local.prow_service_account
 }
 
-// Create a key for GCP Service Account k8s-infra-prow
```

Comment:
This is the key that was revoked by @ameukam recently as discussed privately. Cleanup the key and references to the secrets.